unit/src/main/java/org/apache/falcon/unit/FalconUnitClient.java

```diff
@@ -285,7 +285,16 @@ public APIResult submitExtensionJob(String extensionName, String jobName, String
         InputStream configStream = getServletInputStream(configPath);
         try {
             List<Entity> entities = getEntities(extensionName, jobName, configStream);
```

Comment:
If getEntities can return a Map<EntityType, List<Entity>>. It can be directly passed to submitExtensionJob. Less redundant code.